webui/PCWConfig.py

```diff
@@ -87,6 +87,24 @@ def get_providers_for(feature: str, namespace: str):
         return ConfigFile().getList('{}.namespace.{}/providers'.format(feature, namespace),
                                     ConfigFile().getList('{}/providers'.format(feature), ['EC2', 'AZURE', 'GCE']))
 
+    @staticmethod
+    def get_k8s_providers() -> list:
```

Comment:
we can delete it now

Reply:
Yes, I did yesterday but I didnt' commit the changes :)